lib/matplotlib/pyplot.py

```diff
@@ -2438,10 +2438,12 @@ def matshow(A: ArrayLike, fignum: None | int = None, **kwargs) -> AxesImage:
     """
     Display an array as a matrix in a new figure window.
 
-    The origin is set at the upper left hand corner and rows (first
-    dimension of the array) are displayed horizontally.  The aspect
-    ratio of the figure window is that of the array, unless this would
-    make an excessively short or narrow figure.
+    The origin is set at the upper left hand corner.
+    In the context of a 2D array, the first dimension represents
+    the number of rows(the height of the figure) and the second dimension
+    represents the number of columns(the width of the figure).
+    The aspect ratio of the figure window is that of the array,
+    unless this would make an excessively short or narrow figure.
```

Comment:
```suggestion
    The origin is set at the upper left hand corner.
    In the context of a 2D array, the first dimension represents
    the number of rows (the height of the figure) and the second dimension
    represents the number of columns (the width of the figure).
    The aspect ratio of the figure window is that of the array,
    unless this would make an excessively short or narrow figure.
```